qpdf/qtest/qpdf/qpdfjob-ctest.out

```diff
@@ -9,3 +9,8 @@ json warn test passed
 |custom|qpdfjob json|custom|: |custom|an output file name is required; use - for standard output|custom|
 |custom|qpdfjob json|custom|: |custom|an output file name is required; use - for standard output|custom|
 json error test passed
+qpdfjob: d.pdf: write progress: 0%
+....other write progress....
+qpdfjob: d.pdf: write progress: 100%
+qpdfjob: open m.pdf: No such file or directory
```

Comment:
Is this your intention?

Reply:
This is intentional.

This was entirely clear to me when I wrote it, but with the wisdom that comes from living another 2 weeks I can see that a comment would have been a good idea. Now added.

Reply:
From the wisdom of one more day, I see that what concerned me was the lack of anything that actually looks at d.pdf to make sure it's correct. But, in fact, I actually see that check. So never mind.

Reply:
> This was entirely clear to me when I wrote it, but with the wisdom that comes from living another 2 weeks I can see that a comment would have been a good idea. Now added.

I'm going to merge as is with the comment, which I think adequately resolves the issue. As a habit, I typically call files like this "does-not-exist.pdf" or something similar.

Reply:
> As a habit, I typically call files like this "does-not-exist.pdf" 

That sounds like an obvious solution that should have occurred to me without prompting.